src/raftstore/store/store.rs

```diff
@@ -1774,6 +1775,72 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         };
     }
 
+    fn on_compaction_finished(&mut self, event: CompactedEvent) {
```

Comment:
can this function be called frequently?

Reply:
Each compaction job whose output level > 1 will call this function once, so i don't think it will involve frequently.